rules/docker-eventd.mk

```diff
@@ -21,9 +21,6 @@ $(DOCKER_EVENTD)_INSTALL_DEBS = $(PYTHON3_SWSSCOMMON)
 $(DOCKER_EVENTD)_VERSION = 1.0.0
 $(DOCKER_EVENTD)_PACKAGE_NAME = eventd
 
-$(DOCKER_DHCP)_SERVICE_REQUIRES = updategraph
-$(DOCKER_DHCP)_SERVICE_AFTER = database
```

Comment:
>database [](http://example.com/codeflow?start=31&length=8)

Why remove database?

Reply:
Same issue, we should not modify $(DOCKER_DHCP)_SERVICE_AFTER in this makefile